apps_product_creator/models/odoo_module.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2017-Today: Odoo Community Association (OCA)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import models, fields, api
+
+
+class OdooModule(models.Model):
+    _inherit = 'odoo.module'
+
+    @api.depends(
+        'product_template_id',
+        'product_template_id.product_variant_ids'
```

Comment:
Just a question : if product_template_id.product_variant_ids is set, is 'product_template_id' necessary ?

Reply:
As far as I know, it should be necessary because with only _product_template_id.product_variant_ids_, the method would be triggered only if something changes on the _product.product_ records related on _product_template_id.product_variant_ids_, not if _product_template_id_ itself is edited.

Reply:
OK. Thanks for the precision. I didn't know.